js/kraken.js

```diff
@@ -68,6 +68,7 @@ module.exports = class kraken extends Exchange {
                 'fetchTicker': true,
                 'fetchTickers': true,
                 'fetchTime': true,
+                'fetchStatus': undefined,
```

Comment:
we emulate `fetchStatus` with `fetchTime` so I'm not entirely sure we should set it as undefined.

Reply:
ah, I forgot it, hadn't taken a look.